generators/element/templates/demo/element.story.ejs

```diff
@@ -17,14 +25,28 @@ const defaultContent = tools.autoContent(1, 2);
 
 stories.addDecorator(bridge.withKnobs);
 
+<% if (events.length > 0) { -%>
+// Log events
+<% for(let i = 0; i < events.length; i++) { -%>
+stories.addDecorator(withActions(`${<%= elementClassName %>.tag}:<%= events[i] %>`));
```

Comment:
I'm seeing an `withActions` is not defined error. I don't see it being imported either. 🤔

Reply:
If it helps: 
![image](https://user-images.githubusercontent.com/2671663/117327206-5d479d80-ae60-11eb-89a4-c5a6e484270c.png)

Reply:
You're right! `import { withActions } from "@storybook/addon-actions";`

Reply:
Added it!